stdlib/sys.mli

```diff
@@ -411,7 +411,9 @@ external opaque_identity : 'a -> 'a = "%opaque"
 (** For the purposes of optimization, [opaque_identity] behaves like an
     unknown (and thus possibly side-effecting) function.
 
-    At runtime, [opaque_identity] disappears altogether.
+    At runtime, [opaque_identity] disappears altogether.  However, it does 
+    prevent the argument from being garbage collected until after the location
+    the call appears.
```

Comment:
```suggestion
    At runtime, [opaque_identity] disappears altogether.  However, it does 
    prevent the argument from being garbage collected until the location
    where the call would have appeared.
```
Since the call disappears, "after" isn't particularly meaningful.

Reply:
perhaps "where the call would have occurred" would be even better?  It's odd like you say to use "disappears" and "appeared".  And it's the occurrence we really care about, not it's "appearance".  Let me know and I'll change to whichever you prefer.

Reply:
Yes, "occurred" is better.

Reply:
I have made the change.